lib/plug/redoc_ui.ex

```diff
@@ -50,8 +51,9 @@ defmodule Redoc.Plug.RedocUI do
   def init(opts) do
     spec_url = Keyword.fetch!(opts, :spec_url)
     redoc_version = Keyword.get(opts, :redoc_version, "latest")
+    title = Keyword.get(opts, :title, "ReDoc")
 
-    [spec_url: spec_url, redoc_version: redoc_version]
+    [spec_url: spec_url, redoc_version: redoc_version, title: title]
```

Comment:
I started thinking that spec_url should group to something related to ReDoc tag option only. Maybe named to `redoc_opts` perhaps. :)